idseq_dag/steps/run_srst2.py

```diff
@@ -0,0 +1,34 @@
+import shutil
+
+from idseq_dag.engine.pipeline_step import PipelineStep
+import idseq_dag.util.command as command
+
+class PipelineStepRunSRST2(PipelineStep):
+    '''
+    srst2 is used to detect resistance genes.
+    See: https://github.com/katholt/srst2
+    '''
+    def run(self):
+        ''' Invoking srst2 '''
+        fwd_rd = self.input_files_local[0][0]
+        rev_rd = self.input_files_local[0][1]
+        # TODO: pass in database in additional_files field 
+        db_file_path = self.input_files_local[1][0]
```

Comment:
To switch to using additional_files, you would just write:
```
db_file_path = fetch_from_s3(self.additional_files["resist_gene_db"], self.output_dir_local)
```
You can see from https://github.com/chanzuckerberg/idseq-dag/blob/master/idseq_dag/engine/pipeline_step.py#L94 that self.output_dir_local is where the input files are -- since for SRST2 you wanted to have the reference in the same directory as the inputs.